### ISSUE TITLE
Knives are comedically large as seen in my profile picture.

### ISSUE THREAD

**Issue by WANTEDSHADY** [jasonrohrer/OneLife#1090]
Knives are comedically large as seen in my profile picture.

**Comment by ContractTakingRichHuman** [comment]
I agree, my utility knife is smaller than my hand, why is it so big? 
![image](https://github.com/jasonrohrer/OneLife/assets/171103057/b455890c-1c58-4c8b-8701-8ee0ee54b21c)
(NO DIDDY)

**Comment by GnihtonSeodYdahs** [comment]
That's what she said
